metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -175,6 +175,34 @@ transformers:
 The main use case of `simple_remove_dataset_ownership` is to remove incorrect owners present in the source. You can use it along with the [Simple Add Dataset ownership](#simple-add-dataset-ownership) to remove wrong owners and add the correct ones.
 
 Note that whatever owners you send via `simple_remove_dataset_ownership` will overwrite the owners present in the UI.
+## Extract Dataset globalTags
+### Config Details
+| Field                       | Required | Type         | Default       | Description                                                      |
+|-----------------------------|----------|--------------|---------------|------------------------------------------------------------------|
+| `extract_tags_from`         | ✅       | string       |  `urn`             | Which field to extract tag from. Currently only `urn` is supported.  |
+| `extract_tags_regex`        | ✅       | string       |  `.*`             | Regex to use to extract tag.|
+| `replace_existing`          |          | boolean      | `false`       | Whether to remove owners from entity sent by ingestion source.   |
```

Comment:
isn't `semantics` and `replace_existing` is the same?
I mean I guess I can't set `replace_existing` and set semantics to `OVERWRITE` right?

Reply:
`replace_existing` is what is used in ALL transformers. I am following what is already used in all transformers. https://github.com/search?q=repo%3Adatahub-project%2Fdatahub%20replace_existing&type=code

Reply:
Doing that refactor across all transformers while really good is not really I want to do in this PR. That will increase the testing scope of this PR well outside of the current functionality that I am trying to add here.